modules/tinymce/src/core/main/ts/content/EditorContent.ts

```diff
@@ -5,14 +5,54 @@
  * For commercial licenses see https://www.tiny.cloud/
  */
 
+import { Cell } from '@ephox/katamari';
+
+import { addDefaultGetFormats } from 'tinymce/core/content/GetContentImpl';
+import { addDefaultSetFormats } from 'tinymce/core/content/SetContentImpl';
+
+import Editor from '../api/Editor';
+import * as AddFormat from './AddFormat';
 import { Content, GetContentArgs, SetContentArgs } from './ContentTypes';
 import { getContent } from './GetContent';
 import { setContent } from './SetContent';
 
+interface EditorContent {
+  addFormat: (format: string, formatGetter: GetContentFormatter, formatSetter: SetContentFormatter) => void;
```

Comment:
This name is not per the spec, it should be called `register` format. From the meeting notes:
```
ACTION: change to editor.content.registerFormat
```

- [ ] Fix incorrect API name